### ISSUE TITLE
 Hyper parameters about 'uda_softmax_temp' and 'uda_confidence_thresh'

### ISSUE THREAD

**Issue by Lan1991Xu** [google-research/uda#15]
Hi, Many thanks for sharing the code. Could please tell me when the labelled data is limited, e,p CIFAR-10 250 labels, the hyperparameters of 'uda_softmax_temp' and 'uda_confidence_thresh'. 
Thanks

**Comment by qizhex** [comment]
Hi, we set uda_confidence_thresh to 0.8 and uda_softmax_temp to 0.9 for that case. Thanks!

**Comment by varunnair18** [comment]
Hi @qizhex,

Are those the hyper-parameters that give SOTA performance (i.e. error rate of 8.41%) on 250 labels for CIFAR-10?

Also, what were the hyperparameters of  'uda_softmax_temp' and 'uda_confidence_thresh' used for 500, 1000, 2000, and 4000 labels for CIFAR-10? 

I noticed that in #14, the command to run the SOTA-run for CIFAR-10 on 4000 labels didn't use 'uda_confidence_thresh' to achieve the best results, so I'm curious at what point 'uda_confidence_thresh' no longer provided benefits to the training process.

Thanks!

**Comment by qizhex** [comment]
Hi, we used uda_softmax_temp and uda_confidence_thresh only for 250 labels. When the size of the labeled set is 500, 1000, 2000, we simply use entropy minimization with coefficient  0.1. For CIFAR-10, the entropy should be computed on original unlabeled examples instead of augmented unlabeled examples. For ImageNet, using original examples or augmented examples for entropy minimization leads to similar performance, so in a previous [version](https://github.com/google-research/uda/commit/4175a70b167a2e54db6212e85a2f6ec6f08cf1b0#diff-6cd56132c3094b297316dd2741f47475L360) of the code, the entropy was computed on augmented unlabeled examples.

**Comment by varunnair18** [comment]
Got it, thank you for the information!

**Comment by guotong1988** [comment]
Thank you!
